### PR TITLE
Removing invalid DNS  tld entries from malware.txt

### PR DESCRIPTION
**Pull request by aww-yiss**
Signed-off-by: Josh Bradley <jbradley@deimos.lan>

## Checklist

- [x] I have verified that I have not modified the following files:
      - inside the `adguard` folder
      - inside the `alt-version` folder
      - inside the `dnsmasq-version` folder
      - everything.txt

Because those files will be automatically updated using GitHub Actions!

### CODE REVIEW

File: malware.txt [hunk 1]
```diff
@@ -257463,7 +257462,6 @@
 0.0.0.0 usaa.com.inet.ent.logon.logon.redirectedfromlogoff.truemain.warefpub.globalproducts.priauth.nav.studentsuccess.com.au
 0.0.0.0 usaa.com.inet.ent.logon.logon.redirectjsp.true.registrationaprofile.estatement.myaccountsregistration.vesplast.com
 0.0.0.0 usaa.com.inet.entlogon.logon.redirectjsp.ef4bce064403e276e26b792bda81c384ce09593b819e632a1.3923ef2c8955cooskieid112824nosdeasd.
```
Comment (spirillen) [inline malware.txt:257464]: This line is invalid as well...

File: malware.txt [hunk 2]
```diff
@@ -206748,7 +206748,6 @@
 0.0.0.0 protection.secure.confirmation.fbid1703470323273355seta.1375333179420406.10737418.smktarunabhakti.net
 0.0.0.0 protection.sketchwefair-watduoliprudential.com.watchdogdns.duckdns.org
 0.0.0.0 protection201-account.6827-update.team.com8serv1130.webs001231cr-cm-lgin-submmit-id99-l0gin-submito9-id.ppi11781-lo0gin-loogiin
```
Comment (spirillen) [inline malware.txt:206750]: This line is invalid as well...

**Review by spirillen — CHANGES_REQUESTED** [review]
You need to remove two more records...

File: malware.txt (hunk 1 above)
Comment (aww-yiss) [reply malware.txt]: fixed thanks

File: malware.txt (hunk 2 above)
Comment (aww-yiss) [reply malware.txt]: fixed thanks

**Comment by aww-yiss** [comment]
@spirillen  per your observation, there were several malformed entries in here. I've scrub all the values to check that they at. least end in a valid DNS TLD which has at a minimum some SOA when querying google's 8.8.8.8 resolver.

**Comment by spirillen** [comment]
@aww-yiss for another time, just use [pyfunceble](https://github.com/funilrys/pyfunceble) :smirk: , then you safe your self a lot of work and google spying

it is as simple as 

```bash
pyfunceble --syntax -f $file
```

**Pull request merged**